### DISCUSSION TITLE
Conflicting functions and builds configuration

### DISCUSSION THREAD

**Discussion by KenLee560** [vercel/community#7267] [Help] [answered]
### Question
https://vercel.com/docs/errors/error-list#conflicting-functions-and-builds-configuration
Is there a way to implement **nodejs** of **Other** frameworks with **functions** and **builds** in vercel.json?
![image](https://github.com/vercel/community/assets/112598761/57929efb-d588-417d-b987-87747f1f7e21)
<br /><br />

![image](https://github.com/vercel/community/assets/112598761/9a3f04eb-665a-4dff-9b29-a0d3dff57661)

I could do functions in my vercel.json, which extended to **60s**, but without builds the website is not working properly
![image](https://github.com/vercel/community/assets/112598761/7ea60dbf-6519-4a6b-8b65-8b0347c28b06)
<br /><br />

While...
https://vercel.com/docs/functions/runtimes#max-duration
![image](https://github.com/vercel/community/assets/112598761/04f441da-23d0-463f-9b95-eab247e5e9eb)

I could do builds in my vercel.json, but without functions, which lead me to **10**s timeout
![image](https://github.com/vercel/community/assets/112598761/1b28263e-a55b-49ee-8f5f-84153b0b9169)
<br /><br />

I just wanted to have 60s timeout for my default maximum duration for all of my request

Currently is "**Hobby**" plan, do I need to have at least "**Pro**" plan, in order to do it or I missing something else?

**Comment by swarnava** [comment, marked as answer]
You can have `60s` timeout on [Hobby plan](https://vercel.com/docs/functions/runtimes#max-duration). However, you can only increase it if they are deployed using Serverless function. It doesn't seem like `index.js` is being deployed as Serverless function currently.

**Reply by KenLee560** [reply]
Thanks for your assist, I think I got it working now

[a comment by an automated account is not shown]
